File: problems/1/solution.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */
var twoSum = function(nums, target) {
    for (var i = 0; i < nums.length; i++) {
        for (var j = 0; j < nums.length; j++) {
            if (i !== j && nums[i] + nums[j] === target) {
                return [j, i];
            }
        }
    }
};

File: problems/1/solution.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        unordered_map<int, int> um;
        
        for (int i = 0; i < size(nums); i++) {
            if (um.count(target - nums[i])) {
                return {um[target - nums[i]], i};
            } else {
                um[nums[i]] = i;
            }
        }
        
        return {-1, -1};
    }
};